### ISSUE TITLE
UIR: Add 5L/6L reports for Body Armour and Two Handed Weapons

### ISSUE THREAD

**Issue by trackpete** [trackpete/exiletools-indexer#79]
If the unique is a Body Armour or a Two Handed Weapon, filter the reports to show data for:

* 4L and less
* 5L
* 6L

**Comment by trackpete** [comment]
Given this some more thought and I think I might implement this as another set of nav-tabs.

Once you select a main report, if the item is Body Armour or Two Handed/Bow, you will be also able to select:

* All Links
* 4L and Below
* 5L
* 6L

i.e. `uniques.html#/Talisman/Ashrend%20Buckskin%20Tunic/PastWeek/All` or `uniques.html#/Talisman/Ashrend%20Buckskin%20Tunic/PastWeek/5L` etc.

This will require new ng-routes but should work fluidly within the current tabs searchfilters objects/etc.

**Comment by trackpete** [comment]
I implemented this and it works great - if there are actually appropriate items in the index.

Instead of checking if it's Body Armour/etc., I need to actually check to see if there are items that match with links. Then I need to pay attention to some of the error issues - for example, Ashrend may have some 5L's posted but none are GONE.

**Comment by trackpete** [comment]
I didn't really re-read this and think about it when I started writing some error checking code. The upside is that I've added better problem/error generation into the app. The downside is that users can still choose to run a 6L report and thus get the error.

I think for now I'll just leave it as is. I can go back later and add some more intelligence if it's a problem.

![image](https://cloud.githubusercontent.com/assets/8779751/12588037/d2fd7da0-c424-11e5-9e50-98b9402e8eb6.png)
